main.go

```diff
@@ -3,19 +3,17 @@ package main
 import (
 	"flag"
 	"fmt"
-	"github.com/boourns/scaffold/ast"
-	"github.com/boourns/scaffold/controller"
-	"github.com/boourns/scaffold/model"
-	"github.com/boourns/scaffold/static"
 	"os"
+	"scaffold/ast"
+	"scaffold/model"
+	"scaffold/static"
```

Comment:
I *think* this has to stay `github.com/boourns/scaffold/...`, and the module name in go.mod has to also be the full URL.

I tried installing scaffold via `go get` after the PR was merged, and it was complaining about the module name.

I'm hacking around on master for a bit, with the goal to be able to `go get` this package and have it install cleanly from scratch

Reply:
You're right. Looking at this file https://github.com/gin-gonic/gin/blob/master/auth.go, they've imported `gitHub.com/gin-gonic/gin...` for an internal package.
I made this change assuming that was the reason my fork wasn't building properly (iamkevb instead of boourns).

Reply:
oh right github forks and golang package names dont mix
i just added you as collaborator so you should be able to push to this repo instead of your fork